app/controllers/crawl_bars_controller.rb

```diff
@@ -1,2 +1,24 @@
 class CrawlBarsController < ApplicationController
+
+    def destroy
+        crawlbar = CrawlBar.find_by(crawl_id: params[:id], bar_id: params[:bar_id])
+        debugger
```

Comment:
might want to remove this debugger 🐛

Reply:
Oooh good call. We haven’t actually finished the fetch call for this yet, this is as far as we got. we may need your help with it tomorrow